WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/DaySelectionBuilder.kt

```diff
@@ -25,14 +27,16 @@ class DaySelectionBuilder
 ) {
     fun buildSelection(
         bloggingRemindersModel: BloggingRemindersUiModel?,
-        onSelectDay: (DayOfWeek) -> Unit
+        onSelectDay: (DayOfWeek) -> Unit,
+        onSelectTime: () -> Unit
     ): List<BloggingRemindersItem> {
         val daysOfWeek = daysProvider.getDaysOfWeekByLocale()
         val text = dayLabelUtils.buildNTimesLabel(bloggingRemindersModel)
         val nTimesLabel = MediumEmphasisText(
                 EmphasizedText(text),
                 bloggingRemindersModel?.enabledDays?.isEmpty() == true
         )
+        val time = bloggingRemindersModel?.getNotificationTime() ?: "10:00 AM"
```

Comment:
We shouldn't need to ever set this hardcoded time 🤔

Reply:
Cleaned up, after merging FluxC PR.   Thanks